build.ps1

```diff
@@ -81,11 +81,6 @@ Trace-Log "Build #$BuildNumber started at $startTime"
 
 Test-BuildEnvironment -CI:$CI
 
-if (-not $VSToolsetInstalled) {
```

Comment:
why not check for a toolset anymore?

Reply:
Figured out toolsets are no more. Yay :)

It won't affect my workflow, but worth notifying others of this change.

Reply:
I never understood how the toolset/config file worked anyway. How might it affect other people's workflows? Auto-detecting msbuild on the PATH, and falling back to the latest that vswhere provides seems quite safe to me.

Reply:
Once you run configure.ps1, you'd be tied to that version of msbuild on *every* build.ps1 run regardless where it's coming from. 

So if your workflow is to *not* always use the latest msbuild, you *could* run into some problems. 

Worth running by the team.